README.md

```diff
@@ -1 +1,57 @@
-sfdx-travisci-orgs
+# sfdx-travisci-org [![Build Status](https://travis-ci.org/forcedotcom/sfdx-travisci-org.svg?branch=master)](https://travis-ci.org/forcedotcom/sfdx-travisci-org)
+
+For a fully guided walk through of setting up and configuring this sample, see the [Continuous Integration Using Salesforce DX](https://trailhead.salesforce.com/modules/sfdx_travis_ci) Trailhead module.
+
+This repository shows how to successfully setup deploying to non-scratch orgs (i.e. Sandbox or Production) with Travis CI. We make a few assumptions in this README:
+
+- You know how to get your Github repository setup with Travis CI. (Here's their [Getting Started guide](https://docs.travis-ci.com/user/getting-started/).)
+- You've installed the [Travis CLI](https://github.com/travis-ci/travis.rb#installation). 
+- You have properly setup JWT-Based Authorization Flow (i.e. headless). I recommend using [these steps for generating your Self-Signed SSL Certificate](https://devcenter.heroku.com/articles/ssl-certificate-self). 
+
+If any any of these assumptions aren't true, the following steps won't work.
+
+## Getting Started
+1) [Fork](http://help.github.com/fork-a-repo/) this repo into your github account using the fork link at the top of the page.
+
+2) Clone your forked repo locally: `git clone https://github.com/<git_username>/sfdx-travisci-org.git`
+
+3) Make sure you have the Salesforce CLI installed. Check by running `sfdx force --help` and confirm you see the command output. If you don't have it installed you can download and install it from [here](https://developer.salesforce.com/tools/sfdxcli).
+
+4) Setup a JWT-based auth flow for the target orgs that you want to deploy to.  This example used the same server.crt file for the connected app in the target org.
```

Comment:
I think it'd be worth noting here that this step will result in the creation of a `server.key` file, which is required by subsequent steps.